src/components/Settings/ChangePassword.vue

```diff
@@ -34,7 +34,8 @@
       <q-btn
         color="primary"
         @click="save"
-        loader
+        :disable="!hasNewPassword || !hasOldPassword"
```

Comment:
Please also disable the button if the new password equals the old one (i.e., if the password hashes are equal).

Reply:
@mr-kojo  according to me that should be validated in the backend after submission for security purpose i talked to my mentor(for rgsoc) about this even he suggested to do the same as most of the websites follow. Whatever your suggestions i will proceed accordingly?